### DISCUSSION TITLE
Distill Weighted Ensemble (Tabular Predictor)

### DISCUSSION THREAD

**Discussion by j2schmit** [autogluon/autogluon#928] [Q&A] [answered]
First off, I'm enjoying exploring this software, and the many nice features that have been added. I love the distill method, but I have one question about it with respect to the Tabular Predictor. Is it possible to use the distill method on only the 'best' model (in this case the weighted ensemble)? As I am using it now all of the individual model classes are distilled, rather than selecting a single model to be distilled.

**Comment by Innixma** [comment, marked as answer]
It is using the weighted ensemble as the teacher model, and training students of various model types.

If you'd like to control which students are trained, specify the `hyperparameters` argument, such as `hyperparameters={'NN': {}}` if you just want a neural network student model.

Note that 'WeightedEnsemble' is a stacker model, and therefore requires other models to exist to make predictions for it. I have recently added support in pre-release for `WeightedEnsemble` to also be trained in distillation which may be what you are looking for: #937

**Reply by j2schmit** [reply]
Thanks, that's exactly the info I was looking for.
